midp/gfx.js

```diff
@@ -1619,8 +1648,8 @@ var currentlyFocusedTextEditor;
     var CONTINUOUS_RUNNING = 2;
 
     Native["javax/microedition/lcdui/GaugeLFImpl.createNativeResource0.(ILjava/lang/String;IZII)I"] =
-    function(addr, ownerId, label, layout, interactive, maxValue, initialValue) {
-        if (label !== null) {
+    function(addr, ownerId, labelAddr, layout, interactive, maxValue, initialValue) {
+        if (labelAddr !== null) {
```

Comment:
> This now needs to be compared against the null constant.

adcf2ec updates all these conditionals to compare addresses with the null constant.